src/Sarif.WorkItemFiling/AzureDevOpsClient.cs

```diff
@@ -67,37 +67,40 @@ public override async Task<IEnumerable<WorkItemFilingMetadata>> FileWorkItems(IE
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
-                        Path = $"/fields/{WorkItemFields.Title}",
+                        Path = $"/fields/{AzureDevOpsFieldNames.Title}",
                         Value = metadata.Title
                     },
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
-                        Path = $"/fields/{WorkItemFields.ReproSteps}",
+                        Path = $"/fields/{AzureDevOpsFieldNames.ReproSteps}",
                         Value = metadata.Description
                     },
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
-                        Path = $"/fields/{WorkItemFields.AreaPath}",
-                        Value = metadata.AreaPath
+                        Path = $"/fields/{AzureDevOpsFieldNames.AreaPath}",
+                        Value = metadata.Area
                     },
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
-                        Path = $"/fields/{WorkItemFields.Tags}",
-                        Value = string.Join(",", metadata.GetAllTags())
+                        Path = $"/fields/{AzureDevOpsFieldNames.Tags}",
+                        Value = string.Join(",", metadata.Tags)
                     }
                 };
 
-                foreach (var customField in metadata.CustomFields)
+                if (metadata.CustomFields != null)
                 {
-                    patchDocument.Add(new JsonPatchOperation
+                    foreach (var customField in metadata.CustomFields)
```

Comment:
>var [](start = 29, length = 3)

Not in this PR but would be nice if this is replaced with type since it's not obvious what metadata.CustomFields is from the right side.

Reply:
agreed, fixed.

---
In reply to: [375955418](https://github.com/microsoft/sarif-sdk/pull/1744#discussion_r375955418) [](ancestors = 375955418)